catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -661,9 +668,13 @@ public Response createMetacard(MultipartBody multipartBody, @Context UriInfo req
                     transformer,
                     metacardId);
 
-            Response.ResponseBuilder responseBuilder = Response.ok(content.getInputStream(),
-                    content.getMimeTypeValue());
-            response = responseBuilder.build();
+            try (InputStream inputStream = content.getInputStream()) {
+                Response.ResponseBuilder responseBuilder = Response.ok(inputStream,
+                        content.getMimeTypeValue());
+                response = responseBuilder.build();
+            } catch (IOException e) {
+                LOGGER.debug("Failed to get content in RESTEndpoint");
```

Comment:
The previous code didn't try to catch any exceptions from `content.getInputStream`. The exception would have bubbled up to the calling method. The only difference would be the type of exception-- `IOException` (checked, unfortunately) versus `RuntimeException`. 

If there is no clean way to use the try-with-resources, I'll probably revert my changes. The benefits of using try-with-resources has to balanced against the amount of changes that need to be make to accommodate it.